inst/tests/mergelist.Rraw

```diff
@@ -0,0 +1,75 @@
+require(methods)
+
+if (exists("test.data.table", .GlobalEnv, inherits=FALSE)) {
```

Comment:
We should have a new helper like `test.setup()` to handle this boilerplate now that our suite of *.Rraw files is growing...

Reply:
#7109